### DISCUSSION TITLE
Why is favorites mobile only and disconnected from starred repos?

### DISCUSSION THREAD

**Discussion by jwildeboer** [community/community#4023] [Mobile]
I really like the favorites option as it allows me to jump to repos I'm interested in (right now German Corona app and EU digital vaccination certificate repos). 

I do NOT like that it is mobile-only and I cannot see my favs when at my desktop, using the browser.

Can you folks somehow combine starred repos with favorites? As in make favorites a subset of starred repos and make the favorites box a general feature, not limited to mobile?

That would be awesome!

Jan

**Comment by brianlovin** [comment]
Thanks Jan! Flagged this for our team – we have some ideas to make Favorites even better, and cross-platform consistency would be useful here. Stay tuned 😊
